docs/synthetics/ssl-certs.md

```diff
@@ -0,0 +1,66 @@
+title: SSL Certificate Monitoring
+description: Monitor & get notified on SSL certificate expiry and change. 
+
+Issues with SSL certificates not only cause downtime of APIs and websites but also hurts their reputation. Sematext Synthetics performs the following SSL certificate checks on all the certificates in the chain - the leaf, intermediate, and root certificates.
+
+* Validate the SSL certificates of the API/websites in every run
+* Check for the expiration of the certificates every day and alert you 30, 14, 7 and 3 days before the expiry
+* Check for certificate change every 10 mins and alert you on detecting any changes, with a detailed change report
+
+Sematext Synthetics alerts you on the failure of these checks via the configured monitor alert channels like Email, Slack, Webhooks, etc. Apart from these checks, Synthetics also provides an SSL certificate report, with details of all the certificates in the chain.
+
+Both the HTTP and Browser monitors perform these checks except for Browser monitors with user journey script configured.
+
+## Certificate Validation
+
+For every run of the monitor, Synthetics performs a set of validity checks on SSL certificates sent by the server. Synthetics will mark the monitor status as failing if any of these checks fail. Synthetics will raise a run failure alert with the details of the failure. Self-signed certificates are not supported. Any websites or APIs using self-signed certificates will fail.
+
+The HTTP monitor performs the following checks:
+
+* If there is a valid chain, connecting the leaf certificate of the server to a trusted root
+* Certificate has not expired
+* Certificate is active (if the certificate is not used before its validity starts)
+* Verify name constraints, if any
+* Verify if there are too many intermediate certificates (more than 3 intermediates)
```

Comment:
Then maybe just: Verify if there are more than 3 intermediate certificates

Reply:
Btw. why 3?  Do we anticipate this being a good number for 99% of sites/certs out there?

Reply:
Too many intermediate certificates the SSL error thrown by Go internal validation code.  Not sure why they have set the limit to 3.